lib/fog/aws/requests/auto_scaling/attach_instances.rb

```diff
@@ -0,0 +1,60 @@
+module Fog
+  module AWS
+    class AutoScaling
+      class Real
+        require 'fog/aws/parsers/auto_scaling/basic'
+
+        # Removes one or more instances from the specified Auto Scaling group.
+        #
+        #  cli equiv:
+        # `aws autoscaling attach-instances --instance-ids i-93633f9b --auto-scaling-group-name my-auto-scaling-group`
+        #
+        # ==== Parameters
+        #
+        #   * AutoScalingGroupName<~String> - The name of the Auto Scaling group``
+        #   * 'InstanceIds'<~Array> - The list of Auto Scaling instances to detach.
+        #
+        # ==== See Also
+        #
+        # http://docs.aws.amazon.com/cli/latest/reference/autoscaling/attach-instances.html
```

Comment:
should be http://docs.aws.amazon.com/AutoScaling/latest/APIReference/API_AttachInstances.html